content/commands/flushall/index.md

```diff
@@ -66,6 +66,7 @@ It is possible to use one of the following modifiers to dictate the flushing mod
 
 * An asynchronous `FLUSHALL` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
 * This command does not delete functions.
+* When using Redis Cluster, this command is identical to `FLUSHDB` since a Redis Cluster supports only database zero.
```

Comment:
that's incorrect.
FLUSHALL has some side effects that FLUSHDB does not, and currently even on a single db deployment these still hold.

for instance, FLUSHALL saves an empty RDB file and resets the dirty counter.

Reply:
i'm not certain if we want to fix it and change the behavior of FLUSHDB on a single-db deployments, or document that. but for sure we should avoid stating wrong facts. :stuck_out_tongue_winking_eye:

Reply:
We definitely want to document the current behavior, including all side effects. Otherwise, people would have false assumptions. Oran, please suggest an edit.

Reply:
something like that:?
```
Other than emptying all databases (similar to FLUSHDB), this command can also clear the persistence RDB file, it aborts any snapshot that's in progress, and if the `save` config is enabled, it saves an empty RDB file.
```

Reply:
@dwdougherty I suggest adding this to `FLUSHALL` based on Oran's input.

Reply:
Change made in last commit.